run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem

```diff
@@ -2511,12 +2709,12 @@ Warnings:                    1
 102  ICOADS_SHIP_NO $ROOT/ICOADS_SHIP/v2014-07/ICOADS.generic.1x1.nc NO 2002/1-12/1/0 C xy kg/m2/s NO 1/5 10 1
 )))ICOADS_SHIP
 
-(((HTAP_SHIP
-102 HTAP_SHIP_NO $ROOT/HTAP/v2015-03/EDGAR_HTAP_NO_SHIPS.generic.01x01.nc emi_no 2008-2010/1/1/0 C xy kg/m2/s NO 1/27 10 2
-)))HTAP_SHIP
+(((HTAPv3_SHIP
+102 HTAPv3_NO_SHP $ROOT/HTAPv3/v2022-12/$YYYY/HTAPv3_NO_0.1x0.1_$YYYY.nc NO_SHP 2000-2018/1-12/1/0 C xy kg/m2/s NO 25 10 4
+)))HTAPv3_SHIP
 
 (((CEDSv2_SHIP
-102 CEDS_NO_SHP $ROOT/CEDS/v2021-06/$YYYY/NO-em-anthro_CMIP_CEDS_$YYYY.nc NO_shp 1750-2019/1-12/1/0 C xy kg/m2/s NO 25 10 5
+102 CEDS_NO_SHP $ROOT/CEDS/v2021-06/$YYYY/NO-em-anthro_CMIP_CEDS_$YYYY.nc NO_shp 1750-2010/1-12/1/0 C xy kg/m2/s NO 25 10 5
```

Comment:
I think this should be changed back to 2019. Otherwise, this will cause differences in the benchmark simulation.

Reply:
Now fixed in https://github.com/geoschem/geos-chem/commit/3f69125dbdf6ae180be14d943941683f4c7b9185

https://github.com/geoschem/geos-chem/blob/3f69125dbdf6ae180be14d943941683f4c7b9185/run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem#L2698-L2756